crates/ibc/src/hosts/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod tendermint_helper;
```

Comment:
```suggestion
pub mod tendermint;
```
Since we're in a "hosts" module, I would find just `tendermint` sufficient.

Perhaps we can add a module docstring explaining that this module contains items (helpers? utils?) that hosts can use.

Reply:
[907f6b2](https://github.com/cosmos/ibc-rs/pull/436/commits/907f6b24a815a8f2fa33f4aa702b67d791f25105)